src/base/bittorrent/session.cpp

```diff
@@ -1496,6 +1434,83 @@ void Session::loadLTSettings(lt::settings_pack &settingsPack)
     }
 }
 
+void Session::configureNetInterfaces(lt::settings_pack &settingsPack)
+{
+    if (!m_listenInterfaceConfigured) {
```

Comment:
This redundant level of indention can be eliminated by returning early on the opposite condition.

Reply:
Seems it's a "one-level-up" logic. It should be outside of this method:
```c++
if (!m_listenInterfaceConfigured)
    configureNetInterfaces(settings);
```

Reply:
@glassez I understand what you're saying, but isn't it safer for `configureNetInterfaces()` to do the self-check? Although in this particular case nothing bad will happen if we re-run this function.

Reply:
Do you insist on this change?

Reply:
No.